src/test/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/mapper/induction/UpdateFutureWorkInterestsEntityMapperTest.kt

```diff
@@ -74,8 +73,6 @@ class UpdateFutureWorkInterestsEntityMapperTest {
 
     // Then
     assertThat(existingFutureWorkInterestsEntity).isEqualToIgnoringInternallyManagedFields(expectedEntity)
-    // TODO RR-469 - fix updatedAt timestamp
-    // assertThat(existingFutureWorkInterestsEntity).wasUpdatedAfter(initialUpdatedAt)
```

Comment:
Did you mean to remove the commented out assertion? Or did you mean to uncomment the assertion now its working?